src/test/java/org/kainos/ea/controller/ResponsibilityControllerUnitTest.java

```diff
@@ -0,0 +1,53 @@
+package org.kainos.ea.controller;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.kainos.ea.api.ResponsibilityService;
+import org.kainos.ea.cli.Responsibility;
+import org.kainos.ea.exception.FailedToGetResponsibilitiesException;
+import org.kainos.ea.resources.ResponsibilityController;
+import org.mockito.Mockito;
+
+import javax.ws.rs.core.Response;
+import java.util.Arrays;
+import java.util.List;
+
+public class ResponsibilityControllerUnitTest {
+
+    ResponsibilityService responsibilityService = Mockito.mock(ResponsibilityService.class);
+
+    @Test
+    void getAllResponsibilities_shouldReturn200StatusCodeAndListOfResponsibilities_whenServiceReturnsListOfResponsibilities() throws FailedToGetResponsibilitiesException {
```

Comment:
add unit tests for getResponsibilitiesForJob method as well

Reply:
unit tests added